### ISSUE TITLE
AQL: Strange "/$CLASS$" attribute when returning data

### ISSUE THREAD

**Issue by yampeku** [ehrbase/ehrbase#329]
## Steps to reproduce ## 

Load any OPT (used [signos.opt](https://github.com/ehrbase/ehrbase/files/5317522/Signos_opt.txt) from my other post)
Load any JSON instance (used [signos_json_correct](https://github.com/ehrbase/ehrbase/files/5317573/signos_ok_json.txt from my other post). The JSON returned by the server is the same as the one sent.
Query to obtain composition/s

## Actual result ##

Returned json contains a "/$CLASS$" attribute, containing "Composition"
```
        "archetype_node_id": "openEHR-EHR-COMPOSITION.signos.v1",
        "/$CLASS$": "Composition",
```
It is also worth noticing that returned json also contains correct "_type":"COMPOSITION"

## Expected result ##

No extra attribute is returned

**Comment by chevalleyc** [comment]
Doesn't seem to occur with the latest code base:

```
{
    "q": "SELECT c\n\t\tFROM EHR e\n\t\tCONTAINS COMPOSITION c\n\t\tWHERE c/uid/value = '40240ea9-9136-48a0-99d4-ef267bbe6c60'\n\t\t\n\t\t",
    "columns": [
        {
            "path": "c",
            "name": "#0"
        }
    ],
    "rows": [
        [
            {
                "uid": {
                    "_type": "OBJECT_VERSION_ID",
                    "value": "40240ea9-9136-48a0-99d4-ef267bbe6c60::local.ehrbase.org::1"
                },
                "name": {
                    "_type": "DV_TEXT"
                },
                "_type": "COMPOSITION",
                "context": {
                    "_type": "EVENT_CONTEXT",
                    "setting": {
                        "_type": "DV_CODED_TEXT",
                        "value": "primary nursing care",
                        "defining_code": {
                            "_type": "CODE_PHRASE",
                            "code_string": "229",
                            "terminology_id": {
                                "_type": "TERMINOLOGY_ID",
                                "value": "openehr"
                            }
                        }
                    },
                    "start_time": {
                        "_type": "DV_DATE_TIME",
                        "value": "2020-10-01T18:46:42Asia/Bangkok"
                    }
                },
                "category": {
                    "_type": "DV_CODED_TEXT",
                    "value": "event",
                    "defining_code": {
                        "_type": "CODE_PHRASE",
                        "code_string": "433",
                        "terminology_id": {
                            "_type": "TERMINOLOGY_ID",
                            "value": "openehr"
                        }
                    }
                },
                "composer": {
                    "name": "Dr. Spock",
                    "_type": "PARTY_IDENTIFIED",
                    "external_ref": {
                        "id": {
                            "_type": "HIER_OBJECT_ID",
                            "value": "930f703e-048b-11eb-adc1-0242ac120002"
                        },
                        "type": "PERSON",
                        "_type": "PARTY_REF",
                        "namespace": "DEMOGRAPHIC"
                    }
                },
                "language": {
                    "_type": "CODE_PHRASE",
                    "code_string": "es",
                    "terminology_id": {
                        "_type": "TERMINOLOGY_ID",
                        "value": "ISO_639-1"
                    }
                },
                "territory": {
                    "_type": "CODE_PHRASE",
                    "code_string": "ES",
                    "terminology_id": {
                        "_type": "TERMINOLOGY_ID",
                        "value": "ISO_3166-1"
                    }
                },
                "archetype_details": {
                    "_type": "ARCHETYPED",
                    "rm_version": "1.0.4",
                    "template_id": {
                        "_type": "TEMPLATE_ID",
                        "value": "signos.es.v1"
                    },
                    "archetype_id": {
                        "_type": "ARCHETYPE_ID",
                        "value": "openEHR-EHR-COMPOSITION.signos.v1"
                    }
                },
                "archetype_node_id": "openEHR-EHR-COMPOSITION.signos.v1",
                "content": [
                    {
                        "name": {
                            "value": "Pulso",
                            "_type": "DV_TEXT"
                        },
                        "_type": "OBSERVATION",
                        "subject": {
                            "_type": "PARTY_SELF"
                        },
                        "encoding": {
                            "code_string": "UTF-8",
                            "terminology_id": {
                                "name": "Unicode",
                                "value": "Unicode"
                            }
                        },
                        "language": {
                            "code_string": "es",
                            "terminology_id": {
                                "name": "ISO_639-1",
                                "value": "ISO_639-1"
                            }
                        },
                        "data": {
                            "name": {
                                "value": "*history(en)",
                                "_type": "DV_TEXT"
                            },
                            "origin": {
                                "value": "2020-10-02T08:46:42",
                                "_type": "DV_DATE_TIME"
                            },
                            "archetype_node_id": "at0002",
                            "events": [
                                {
                                    "name": {
                                        "value": "*Any event(en)",
                                        "_type": "DV_TEXT"
                                    },
                                    "time": {
                                        "value": "2020-10-02T08:46:42",
                                        "_type": "DV_DATE_TIME"
                                    },
                                    "_type": "POINT_EVENT",
                                    "data": {
                                        "name": {
                                            "value": "*structure(en)",
                                            "_type": "DV_TEXT"
                                        },
                                        "archetype_node_id": "at0001",
                                        "_type": "ITEM_TREE",
                                        "items": [
                                            {
                                                "name": {
                                                    "value": "Frecuencia del pulso",
                                                    "_type": "DV_CODED_TEXT",
                                                    "defining_code": {
                                                        "code_string": "at1026",
                                                        "terminology_id": {
                                                            "value": "local"
                                                        }
                                                    }
                                                },
                                                "value": {
                                                    "units": "/min",
                                                    "magnitude": 70,
                                                    "_type": "DV_QUANTITY"
                                                },
                                                "_type": "ELEMENT",
                                                "archetype_node_id": "at0004"
                                            }
                                        ]
                                    },
                                    "archetype_node_id": "at0003"
                                }
                            ],
                            "_type": "HISTORY"
                        },
                        "archetype_node_id": "openEHR-EHR-OBSERVATION.pulse.v1"
                    },
                    {
                        "name": {
                            "value": "Height/Length",
                            "_type": "DV_TEXT"
                        },
                        "_type": "OBSERVATION",
                        "subject": {
                            "_type": "PARTY_SELF"
                        },
                        "encoding": {
                            "code_string": "UTF-8",
                            "terminology_id": {
                                "name": "Unicode",
                                "value": "Unicode"
                            }
                        },
                        "language": {
                            "code_string": "es",
                            "terminology_id": {
                                "name": "ISO_639-1",
                                "value": "ISO_639-1"
                            }
                        },
                        "data": {
                            "name": {
                                "value": "history",
                                "_type": "DV_TEXT"
                            },
                            "origin": {
                                "value": "2020-10-02T08:46:42",
                                "_type": "DV_DATE_TIME"
                            },
                            "archetype_node_id": "at0001",
                            "events": [
                                {
                                    "name": {
                                        "value": "Any event",
                                        "_type": "DV_TEXT"
                                    },
                                    "time": {
                                        "value": "2020-10-02T08:46:42",
                                        "_type": "DV_DATE_TIME"
                                    },
                                    "_type": "POINT_EVENT",
                                    "data": {
                                        "name": {
                                            "value": "Simple",
                                            "_type": "DV_TEXT"
                                        },
                                        "archetype_node_id": "at0003",
                                        "_type": "ITEM_TREE",
                                        "items": [
                                            {
                                                "name": {
                                                    "value": "Height/Length",
                                                    "_type": "DV_TEXT"
                                                },
                                                "value": {
                                                    "units": "cm",
                                                    "magnitude": 186,
                                                    "_type": "DV_QUANTITY"
                                                },
                                                "_type": "ELEMENT",
                                                "archetype_node_id": "at0004"
                                            }
                                        ]
                                    },
                                    "archetype_node_id": "at0002"
                                }
                            ],
                            "_type": "HISTORY"
                        },
                        "archetype_node_id": "openEHR-EHR-OBSERVATION.height.v1"
                    },
                    {
                        "name": {
                            "value": "Peso corporal",
                            "_type": "DV_TEXT"
                        },
                        "_type": "OBSERVATION",
                        "subject": {
                            "_type": "PARTY_SELF"
                        },
                        "encoding": {
                            "code_string": "UTF-8",
                            "terminology_id": {
                                "name": "Unicode",
                                "value": "Unicode"
                            }
                        },
                        "language": {
                            "code_string": "es",
                            "terminology_id": {
                                "name": "ISO_639-1",
                                "value": "ISO_639-1"
                            }
                        },
                        "data": {
                            "name": {
                                "value": "*history(en)",
                                "_type": "DV_TEXT"
                            },
                            "origin": {
                                "value": "2020-10-02T08:46:42",
                                "_type": "DV_DATE_TIME"
                            },
                            "archetype_node_id": "at0002",
                            "events": [
                                {
                                    "name": {
                                        "value": "Cualquier evento.",
                                        "_type": "DV_TEXT"
                                    },
                                    "time": {
                                        "value": "2020-10-02T08:46:42",
                                        "_type": "DV_DATE_TIME"
                                    },
                                    "_type": "POINT_EVENT",
                                    "data": {
                                        "name": {
                                            "value": "*Simple(en)",
                                            "_type": "DV_TEXT"
                                        },
                                        "archetype_node_id": "at0001",
                                        "_type": "ITEM_TREE",
                                        "items": [
                                            {
                                                "name": {
                                                    "value": "Peso",
                                                    "_type": "DV_TEXT"
                                                },
                                                "value": {
                                                    "units": "kg",
                                                    "magnitude": 98,
                                                    "_type": "DV_QUANTITY"
                                                },
                                                "_type": "ELEMENT",
                                                "archetype_node_id": "at0004"
                                            }
                                        ]
                                    },
                                    "archetype_node_id": "at0003"
                                }
                            ],
                            "_type": "HISTORY"
                        },
                        "archetype_node_id": "openEHR-EHR-OBSERVATION.body_weight.v1"
                    },
                    {
                        "name": {
                            "value": "Respirations",
                            "_type": "DV_TEXT"
                        },
                        "_type": "OBSERVATION",
                        "subject": {
                            "_type": "PARTY_SELF"
                        },
                        "encoding": {
                            "code_string": "UTF-8",
                            "terminology_id": {
                                "name": "Unicode",
                                "value": "Unicode"
                            }
                        },
                        "language": {
                            "code_string": "es",
                            "terminology_id": {
                                "name": "ISO_639-1",
                                "value": "ISO_639-1"
                            }
                        },
                        "data": {
                            "name": {
                                "value": "history",
                                "_type": "DV_TEXT"
                            },
                            "origin": {
                                "value": "2020-10-02T08:46:42",
                                "_type": "DV_DATE_TIME"
                            },
                            "archetype_node_id": "at0001",
                            "events": [
                                {
                                    "name": {
                                        "value": "Any event",
                                        "_type": "DV_TEXT"
                                    },
                                    "time": {
                                        "value": "2020-10-02T08:46:42",
                                        "_type": "DV_DATE_TIME"
                                    },
                                    "_type": "POINT_EVENT",
                                    "data": {
                                        "name": {
                                            "value": "List",
                                            "_type": "DV_TEXT"
                                        },
                                        "archetype_node_id": "at0003",
                                        "_type": "ITEM_TREE",
                                        "items": [
                                            {
                                                "name": {
                                                    "value": "Rate",
                                                    "_type": "DV_TEXT"
                                                },
                                                "value": {
                                                    "units": "/min",
                                                    "magnitude": 40,
                                                    "_type": "DV_QUANTITY"
                                                },
                                                "_type": "ELEMENT",
                                                "archetype_node_id": "at0004"
                                            }
                                        ]
                                    },
                                    "archetype_node_id": "at0002"
                                }
                            ],
                            "_type": "HISTORY"
                        },
                        "archetype_node_id": "openEHR-EHR-OBSERVATION.respiration.v1"
                    },
                    {
                        "name": {
                            "value": "Blood Pressure",
                            "_type": "DV_TEXT"
                        },
                        "_type": "OBSERVATION",
                        "subject": {
                            "_type": "PARTY_SELF"
                        },
                        "encoding": {
                            "code_string": "UTF-8",
                            "terminology_id": {
                                "name": "Unicode",
                                "value": "Unicode"
                            }
                        },
                        "language": {
                            "code_string": "es",
                            "terminology_id": {
                                "name": "ISO_639-1",
                                "value": "ISO_639-1"
                            }
                        },
                        "data": {
                            "name": {
                                "value": "history",
                                "_type": "DV_TEXT"
                            },
                            "origin": {
                                "value": "2020-10-02T08:46:42",
                                "_type": "DV_DATE_TIME"
                            },
                            "archetype_node_id": "at0001",
                            "events": [
                                {
                                    "name": {
                                        "value": "any event",
                                        "_type": "DV_TEXT"
                                    },
                                    "time": {
                                        "value": "2020-10-02T08:46:42",
                                        "_type": "DV_DATE_TIME"
                                    },
                                    "_type": "POINT_EVENT",
                                    "data": {
                                        "name": {
                                            "value": "blood pressure",
                                            "_type": "DV_TEXT"
                                        },
                                        "archetype_node_id": "at0003",
                                        "_type": "ITEM_TREE",
                                        "items": [
                                            {
                                                "name": {
                                                    "value": "Systolic",
                                                    "_type": "DV_TEXT"
                                                },
                                                "value": {
                                                    "units": "mm[Hg]",
                                                    "magnitude": 120,
                                                    "_type": "DV_QUANTITY"
                                                },
                                                "_type": "ELEMENT",
                                                "archetype_node_id": "at0004"
                                            },
                                            {
                                                "name": {
                                                    "value": "Diastolic",
                                                    "_type": "DV_TEXT"
                                                },
                                                "value": {
                                                    "units": "mm[Hg]",
                                                    "magnitude": 80,
                                                    "_type": "DV_QUANTITY"
                                                },
                                                "_type": "ELEMENT",
                                                "archetype_node_id": "at0005"
                                            }
                                        ]
                                    },
                                    "archetype_node_id": "at0006"
                                }
                            ],
                            "_type": "HISTORY"
                        },
                        "archetype_node_id": "openEHR-EHR-OBSERVATION.blood_pressure.v1"
                    },
                    {
                        "name": {
                            "value": "Body temperature",
                            "_type": "DV_TEXT"
                        },
                        "_type": "OBSERVATION",
                        "subject": {
                            "_type": "PARTY_SELF"
                        },
                        "encoding": {
                            "code_string": "UTF-8",
                            "terminology_id": {
                                "name": "Unicode",
                                "value": "Unicode"
                            }
                        },
                        "language": {
                            "code_string": "es",
                            "terminology_id": {
                                "name": "ISO_639-1",
                                "value": "ISO_639-1"
                            }
                        },
                        "data": {
                            "name": {
                                "value": "History",
                                "_type": "DV_TEXT"
                            },
                            "origin": {
                                "value": "2020-10-02T08:46:42",
                                "_type": "DV_DATE_TIME"
                            },
                            "archetype_node_id": "at0002",
                            "events": [
                                {
                                    "name": {
                                        "value": "Any event",
                                        "_type": "DV_TEXT"
                                    },
                                    "time": {
                                        "value": "2020-10-02T08:46:42",
                                        "_type": "DV_DATE_TIME"
                                    },
                                    "_type": "POINT_EVENT",
                                    "data": {
                                        "name": {
                                            "value": "Tree",
                                            "_type": "DV_TEXT"
                                        },
                                        "archetype_node_id": "at0001",
                                        "_type": "ITEM_TREE",
                                        "items": [
                                            {
                                                "name": {
                                                    "value": "Temperature",
                                                    "_type": "DV_TEXT"
                                                },
                                                "value": {
                                                    "units": "°C",
                                                    "magnitude": 25,
                                                    "_type": "DV_QUANTITY"
                                                },
                                                "_type": "ELEMENT",
                                                "archetype_node_id": "at0004"
                                            }
                                        ]
                                    },
                                    "archetype_node_id": "at0003"
                                }
                            ],
                            "_type": "HISTORY"
                        },
                        "archetype_node_id": "openEHR-EHR-OBSERVATION.body_temperature.v1"
                    }
                ]
            }
        ]
    ]
}
```

**Comment by yampeku** [comment]
Can confirm, seems fixed in 0.14.0
